czishrink/netczicompressTests/Models/ProgramNameAndVersionTests.cs

```diff
@@ -39,7 +38,6 @@ public void Version_WhenCalled_ReturnsExpected()
         var actual = new ProgramNameAndVersion().Version;
 
         // ASSERT
-        var re = new Regex(@"^1\.0\.0(\+\d+)?$");
-        re.IsMatch(actual).Should().BeTrue();
+        actual.Should().MatchRegex(@"^1\.0\.\d+(\+\d+)?$");
```

Comment:
What about `@"^[1-9]+\.[0-9]+\.\d+(\+\d+)?$"` to avoid updating this string for minor/major updates?

Reply:
https://regex101.com/r/8iQEDS/1

Reply:
Decided to keep as is by majority vote.